gensim/models/hdpmodel.py

```diff
@@ -80,65 +122,125 @@ def lda_e_step(doc_word_ids, doc_word_counts, alpha, beta, max_iter=100):
         expElogtheta = np.exp(Elogtheta)
         phinorm = np.dot(expElogtheta, betad) + 1e-100
         meanchange = np.mean(abs(gamma - lastgamma))
-        if (meanchange < meanchangethresh):
+        if meanchange < meanchangethresh:
             break
 
     likelihood = np.sum(counts * np.log(phinorm))
     likelihood += np.sum((alpha - gamma) * Elogtheta)
     likelihood += np.sum(gammaln(gamma) - gammaln(alpha))
     likelihood += gammaln(np.sum(alpha)) - gammaln(np.sum(gamma))
 
-    return (likelihood, gamma)
+    return likelihood, gamma
 
 
 class SuffStats(object):
+    """Stores suff stats for document(s)."""
```

Comment:
what is "suff stats"?  please describe it

Reply:
Is this fine - `"Stores sufficient statistics for the current chunk of document(s) whenever  Hdp model is updated with new corpus. These stats are used when updating lambda and top level sticks. The statistics include number of documents in the chunk, length of words in the documents and top level truncation level."`

Reply:
@gyanesh-m yes :+1: